### PR TITLE
Synchronise master to develop branch

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | A lot of commits have been done directly on the master branch instead of develop.
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is green
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,8 +13,11 @@
     "license": "MIT",
     "authors": [
         {
+            "name": "PrestaShop SA",
+            "email": "contact@prestashop.com"
+        },{
```
Comment (Progi1984) [inline composer.json:18]:
```suggestion
        },
        {
```

Comment (PierreRambaud) [reply composer.json:18]: It's a merge request should have be done ealier in a previous PR

File: tests/DecimalNumberTest.php [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
 use PrestaShop\Decimal\Operation\Rounding;
 
-class NumberTest extends \PHPUnit_Framework_TestCase
+class DecimalNumberTest extends \PHPUnit_Framework_TestCase
 {
 
     /**
      * Given a valid number in a string
-     * When constructing a DecimalNumber with it
+     * When constructing a Number with it
```
Comment (Progi1984) [inline tests/DecimalNumberTest.php:19]: Why changing ?

Comment (PierreRambaud) [reply tests/DecimalNumberTest.php:19]: It's a merge request should have be done ealier in a previous PR

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**